### PR TITLE
Adding a recipe for SOXS

### PR DESCRIPTION
**Pull request by jzuhone**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jzuhone** [comment]
@conda-forge-admin, please ping conda-forge/help-python-c

File: recipes/soxs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    - python
+    - numpy
+    - pip
+    - cython >=0.29.21
+  run:
+    - python
+    - astropy >=3.0
+    - tqdm
+    - pooch
+    - setuptools-scm
```
Comment (xhochy) [inline recipes/soxs/meta.yaml:30]: This should normally be not needed at runtime.

Comment (jzuhone) [reply recipes/soxs/meta.yaml]: Hi @xhochy this is fixed now.

File: recipes/soxs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - numpy
+    - pip
+    - cython >=0.29.21
```
Comment (carterbox) [inline recipes/soxs/meta.yaml:24]:
```suggestion
  host:
    - python
    - numpy
    - pip
    - cython >=0.29.21
    - setuptools >=45
    - setuptools_scm >=6.2
    - toml
    - wheel >=0.36.2
```

File: recipes/soxs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - pyyaml
+    - regions
+    - appdirs
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - soxs
+  commands:
+    - pip check
```
Comment (carterbox) [inline recipes/soxs/meta.yaml:41]:
```suggestion
  commands:
    - pip check
    - get_instrument_data --help
```
Test that your scripts are installed correctly!

File: recipes/soxs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/soxs-{{ version }}.tar.gz
+  sha256: 924a88fa1c5acf7611a0888e86ce6bf7972c9801a26fa3e04a1657eb961a88d2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win]
```
Comment (carterbox) [inline recipes/soxs/meta.yaml:15]: Why skip Windows?

```suggestion
```

**Review by carterbox — CHANGES_REQUESTED** [review]

**Pull request merged**